cpp/src/fil/fil.cu

```diff
@@ -447,6 +448,44 @@ void tl2fil_common(forest_params_t* params, const tl::Model& model,
   params->depth = depth;
 }
 
+void tl2fil_common_multi(forest_params_t* params, const tl::Model& model,
```

Comment:
This essentially duplicates the `tl2fil_common` function above. I think it is a reason to merge treelite models outside of FIL.